### ISSUE TITLE
Examine staged data for 2018-03-16

### ISSUE THREAD

**Issue by mwcraig** [feder-observatory/processed_images#309]
Click [here](https://github.com/feder-observatory/processed_images/edit/master/nights/2018-03-16-README.md) to edit README for this night
Raw images for this night at: http://physics.mnstate.edu/feder_gallery/2018-03-16

This directory seems to need these things:
+ NEEDS_ASTROMETRY.txt
+ NEEDS_OBJECT_NAME.txt

**Comment by mwcraig** [comment]
## NEEDS_ASTROMETRY.txt
```File
M102-001.fit
M102-002.fit
M102-003.fit
M102-004.fit
M51-001B.fit
M51-001R.fit
M67-0001B.fit
M67-0001R.fit
M67-0001V.fit
M67-0002B.fit
M67-0002R.fit
M67-0002V.fit
M67-0003B.fit
M67-0003R.fit
M67-0003V.fit
M67-0004B.fit
M67-0004R.fit
M67-0004V.fit
M67-0005B.fit
M67-0005R.fit
M67-0005V.fit
M81-001B.fit
M81-001R.fit
M82-0003B.fit
M82-0003R.fit
M82-0003V.fit
M82-0004B.fit
M82-0004R.fit
M82-0004V.fit
M82-0005B.fit
M82-0005R.fit
M82-0005V.fit
M82-001B.fit
M82-001R.fit
M82-002R.fit
orion-nebula-0001B.fit
orion-nebula-0001R.fit
orion-nebula-0001V.fit
orion-nebula-0002B.fit
orion-nebula-0002R.fit
orion-nebula-0002V.fit
orion-nebula-0003B.fit
orion-nebula-0003R.fit
orion-nebula-0003V.fit
```

**Comment by mwcraig** [comment]
## NEEDS_OBJECT_NAME.txt
```File
M102-001.fit
M102-002.fit
M102-003.fit
M102-004.fit
```

**Comment by mwcraig** [comment]
The object name and astrometry issues have been fixed.

**Comment by mwcraig** [comment]
Closed by #315
